doc/content/guides/connecting-gateways/multitechconduit/_index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "MultiTech Conduit AEP"
+description: ""
+weight: 8
+---
+
+The [MultiTech MultiConnect® Conduit™](http://www.multitech.net/developer/products/multiconnect-Conduit-platform/) is a configurable, scalable cellular communications gateway for industrial IoT applications. The technical specifications of this Conduit can be found in the software guide in the [official documentation](http://www.multitech.net/developer/products/multiconnect-conduit-platform/conduit/) page.
+
+This guide will help you set up the Multitech Conduit gateway to communicate over {{%tts%}}.
+<!--more-->
+{{< figure src="001_Multitech_Conduit_AEP.png" alt="MultiTech MultiConnect Conduit AEP Gateway" >}}
+
+## Prerequisites
+
+1. User account on {{% tts %}} with rights to create Gateways.
+2. Multitech Conduit AEP Gateway connected to the internet and running the [latest firmware](http://www.multitech.net/developer/downloads/).
+
+> This article addresses the minimum configuration required to connect your Multitech Conduit AEP model to {{% tts %}}.
+> For the first-time hardware and software setup, you can refer to the appropriate configuration guide on the [Multitech documentation](http://www.multitech.net/developer/products/multiconnect-conduit-platform/conduit/) page.
+
+## Registration
+
+Create a gateway by following the instructions for the [Console]({{< ref "/guides/getting-started/console#create-gateway" >}}) or the [CLI]({{< ref "/guides/getting-started/cli#create-gateway" >}}).
+
+> Note: The gateway EUI can be found at the bottom of the gateway under the field &quot;NODE LORA&quot;.
+
+## Configuration using a Terminal
+
+Firstly, you will need to generate a `global_conf.json` file required to add a custom channel plan to your gateway. Follow the steps below to generate the required json file.
+
+> **Warning:** The manual configuration of the frequency plan - required to make it work on the The Things (Enterprise) Stack - may result in a mismatch for some of the frequency bands used for the Multitech Gateway.
```

Comment:
Isn't this the other way around? Using the default frequency plan provided by the gateway will cause a mismatch. We are using a custom frequency plan to specifically prevent this mismatch correct?

Reply:
@KrishnaIyer, Multitech has their own pre-defined frequency plans for the gateway and we're generating a custom frequency plan via the GCS.
The warning was meant to convey that we can't guarantee that the generated frequency plan will work perfectly with the gateway.
Would like to know your inputs on the same.